esp32_s3_korvo_2/esp32_s3_korvo_2.c

```diff
@@ -589,6 +603,46 @@ bool bsp_button_get(const bsp_button_t btn)
     return false;
 }
 
+static uint8_t bsp_get_main_button(void *param)
+{
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+    return (uint8_t)bsp_button_get(BSP_BUTTON_MAIN);
```

Comment:
Is it good to use deprecated function? I think, it will be removed in future, or not?

Reply:
Fixed